geomstats/geometry/embedded_manifold.py

```diff
@@ -55,7 +55,25 @@ def projection(self, point):
         raise NotImplementedError(
             'projection is not implemented.')
 
-    def projection_to_tangent_space(self, vector, base_point):
+    def is_tangent(self, vector, base_point=None):
```

Comment:
is it necessary to add it here, as it is inherited from the parent class Manifold?

Reply:
Good point, I removed it.